### PR TITLE
added debian and fedora names of packages needed for homer_robot_face

### PR DESCRIPTION
**Pull request by niyawe**

### CODE REVIEW

**Review by dirk-thomas — APPROVED** [review]

File: rosdep/base.yaml [hunk 1]
```diff
@@ -1400,13 +1401,15 @@ libdrm-dev:
...
 libesd0-dev:
+  debian: [libesd0-dev]
   fedora: [esound-devel]
   ubuntu: [libesd0-dev]
 libespeak-dev:
   debian: [libespeak-dev]
   fedora: [espeak-devel]
   ubuntu: [libespeak-dev]
 libestools-dev:
+  debian: [libestools2.1-dev]
```
Comment (mikaelarguedas) [inline rosdep/base.yaml]: rosdep doesn't support pure virtual packages, can you use the parent package  `libestools-dev` (https://packages.debian.org/stretch/libestools-dev) instead ?

File: rosdep/base.yaml [hunk 2]
```diff
@@ -2481,6 +2485,8 @@ libqt5-widgets:
   slackware: [qt5]
   ubuntu: [libqt5widgets5]
 libqt5x11extras5-dev:
+  debian: [libqt5x11extras5-dev]
+  fedora: [libQt5X11Extras-devel]
```
Comment (mikaelarguedas) [inline rosdep/base.yaml]: I think that we use only RPM available on the Fedora package database. Maybe @wjwwood or @rmattes can confirn this

File: rosdep/base.yaml (hunk 1 above)
Comment (niyawe) [inline rosdep/base.yaml]: in debian jessie libestools-dev is a virtual package

Comment (mikaelarguedas) [inline rosdep/base.yaml]: In this case we will need to make a subcategory for jessie and for stretch referring to the right non-virtual packages

File: rosdep/base.yaml (hunk 2 above)
Comment (niyawe) [inline rosdep/base.yaml]: It seems like i used the wrong package name. Now I found qt5-qtx11extras-devel (https://apps.fedoraproject.org/packages/qt5-qtx11extras-devel).

**Comment by mikaelarguedas** [comment]
fixed in https://github.com/ros/rosdistro/pull/14464

**Pull request merged**